### PR TITLE
Changed the way skins are retrived

### PR DESCRIPTION
**Pull request by PolarizedIons**
Getting skins via minecraft.net instead of Amazon's s3 because of (very) strong caching on their side. Also safe for the future if the skin location changes.

Also added some comments to explain processes. 

I'll look into caching the API calls to stop rate-limiting as well as limit remote requests.

### CODE REVIEW

**Comment by blha303** [comment]
Honestly, while mcsanta was fun to make, and it was cool to see that people were interested in it, it was always buggy, and I hate PHP. Apparently I never pushed the actual working code to github either, because pulling your repo to my server to test it crashed Apache, and the changes you've made shouldn't have done that. The version on the server works fine, PHP is kinda crazy scary so I'm not going near it, but I will push the updated mcsanta to github so everyone can see how much of a terrible hack it is :P

In short, I think I'm done with mcsanta. Kinda surprised it's been a year already, I remember doing this at this time last year and people were suggesting adding zombie skins for Halloween and stuff. I'd be down for that if anyone could come up with a decent skin mask to add bites or whatever, but nobody did.

**Comment by blha303** [comment]
Also, the upcoming commit will break your PR's mergability.
